From f94fa19769f9b3ee5e1fe5f5afae9a4767856fe7 Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Sat, 2 Nov 2024 22:25:37 +0800
Subject: [PATCH 249/385] AOSCOS: platform/x86: hp-wmi: Mark 8BAB board for
 OMEN thermal profile

Similar to 3a057bf30e044a51af8e6a8fe8cbfac49e2b9bc5 ("platform/x86:
hp-wmi: Add thermal profile support for 8BAD boards"), the HP OMEN 9 2023
(8BAB) board (China-specific model) should also be added to the list of
boards handled by the OMEN thermal profiles.

Signed-off-by: Mingcong Bai <jeffbai@aosc.io>

Link: https://t.me/c/1109254909/613200
Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
Signed-off-by: Mingcong Bai <jeffbai@aosc.io>

[Mingcong Bai: (6.19) Resolved minor merge conflicts in
 drivers/platform/x86/hp/hp-wmi.c]

Signed-off-by: Mingcong Bai <jeffbai@aosc.io>
---
 drivers/platform/x86/hp/hp-wmi.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/platform/x86/hp/hp-wmi.c b/drivers/platform/x86/hp/hp-wmi.c
index f4ea1ea05997b..61940e51a84dd 100644
--- a/drivers/platform/x86/hp/hp-wmi.c
+++ b/drivers/platform/x86/hp/hp-wmi.c
@@ -72,7 +72,7 @@ static const char * const omen_thermal_profile_boards[] = {
 	"88F7", "88FD", "88FE", "88FF",
 	"8900", "8901", "8902", "8912", "8917", "8918", "8949", "894A", "89EB",
 	"8A15", "8A42",
-	"8BAD",
+	"8BAB", "8BAD"
 };
 
 /* DMI Board names of Omen laptops that are specifically set to be thermal
@@ -90,7 +90,7 @@ static const char * const omen_thermal_profile_force_v0_boards[] = {
  */
 static const char * const omen_timed_thermal_profile_boards[] = {
 	"8A15", "8A42",
-	"8BAD",
+	"8BAB", "8BAD",
 };
 
 /* DMI Board names of Victus 16-d1xxx laptops */
-- 
2.52.0

